include/gz/sensors/CameraSensor.hh

```diff
@@ -22,6 +22,7 @@
 #include <string>
 
 #include <gz/msgs/image.pb.h>
+#include <gz/msgs/boolean.pb.h>
```

Comment:
alphabetize

Reply:
Addressed in [d7bc027](https://github.com/gazebosim/gz-sensors/pull/322/commits/d7bc027df336296e9e27c52272e98b4154faf691)